Update paragraph on install-required-packages (#8347)
netdata/netdata: packaging/installer/methods/manual.md

## Prepare your system

> Use our automatic requirements installer (_no need to be `root`_). This will try to find the packages that

Reviewer: ```suggestion
Use our automatic requirements installer (_no need to be `root`_), which attempts to find the packages that
```
